DirectEmployers/MyJobs
# Remove unused modules and sync up shrinkwrap

I used a utility called depcheck to find that we are installing but not using a lot of npm modules. Having these around was causing some troubles with dependency checking, so good riddance. AFAICT everything left in the `package.json` list is here for a reason now.

## Review

* Am I being too aggressive? Should something be put back?
* Is removing `es3ify-loader` going to kill IE8 somewhere important? @fezick says we only need v1 templates to continue working on IE8.

## Test

* `rm -rf gulp/node_modules`
* `dkg npm install`
* Restart devserver and/or disable devserver and run `dkgg` to do a production build.
* Check that js bundled apps and bundled css still works.
